README.rst

```diff
@@ -4,54 +4,55 @@ FreezeFrog
 .. image:: https://circleci.com/gh/closeio/freezefrog/tree/master.svg?style=svg&circle-token=010565a97316df8a248f0f32d584357021a3873b
     :target: https://circleci.com/gh/closeio/freezefrog/tree/master
 
-*FreezeFrog* lets you efficiently mock datetimes in unit tests.
+*FreezeFrog* lets you mock datetimes in tests.
 
-
-(Interested in working on projects like this? `Close.io`_ is looking for `great engineers`_ to join our team)
+(Interested in working on projects like this? `Close.io`_ is looking for `great engineers`_ to join our team.)
 
 .. _Close.io: http://close.io
 .. _great engineers: http://jobs.close.io
 
-
 .. contents:: Contents
 
-
 Why FreezeFrog?
 ---------------
 
-FreezeFrog is a Python library that lets you mock datetimes in unit tests. Its
-goal is to be simple and fast.
+FreezeFrog is a Python library that lets you mock datetimes in tests. Its goal
+is to be simple and fast.
 
-* In comparison to certain other time freezing libraries, FreezeFrog doesn't loop
-  through all imported modules, making it fast even for larger projects.
+* In comparison to certain other time freezing libraries, FreezeFrog doesn't
+  loop through all imported modules, making it fast even for larger projects.
 
 * FreezeFrog currently supports mocking the following basic methods:
 
-  * ``datetime.datetime.now`` (if ``tz_delta`` is specified)
+  * ``datetime.datetime.now``
 
   * ``datetime.datetime.utcnow``
 
   * ``time.time``
 
-
 Usage
 -----
 
 Use the ``FreezeTime`` context manager to freeze the time. Pass the desired
-``datetime`` object to the constructor. The constructor also takes the ``tick``
-keyword argument (``False`` by default), which makes the clock start ticking.
+``datetime`` object to the constructor, and the timezone to mock the system's
+timezone. The constructor also takes the ``tick`` argument (``False`` by
+default), which makes the clock start ticking, and the ``fold`` argument, which
```

Comment:
Does `fold` have a default or would it raise an `AmbiguousTimeError`? (Should probably mention it here)

Reply:
Done in https://github.com/closeio/freezefrog/pull/13/commits/d6ab6863cfb8a80f5c7fe97899bdc3a6f102b19e.